### DISCUSSION TITLE
[idea] visualize dependencies

### DISCUSSION THREAD

**Discussion by loynoir** [verdaccio/verdaccio#2684] [Ideas]
**Is your feature request related to a problem?**
Would be nice to see `dependencies graph`. 😃

Especially, there are local packages concept within `verdaccio`.

**Describe the solution you'd like**
- anvaka
https://npm.anvaka.com/#/view/2d/pino
- source
https://github.com/anvaka/npmgraph.an
- API
https://github.com/anvaka/npmgraph.an/blob/87b8be6d5d49deb150e3794e0286e932a6703eaf/src/scripts/config.js#L2

**Describe alternatives you've considered**

**Additional context**

**Comment by juanpicado** [comment]
This could be added to the UI somehow, more ideas or PoC are welcome.

**Comment by loynoir** [comment]
Some related links

`verdaccio/verdaccio`: Dependents tab

https://github.com/verdaccio/verdaccio/issues/1436

`anvaka/npmgraph.an`: Specify a local npm registry to be used with npmgraph

https://github.com/anvaka/npmgraph.an/issues/38

**Comment by juanpicado** [comment]
This one is really cool and in React 😉 ..... https://npmgraph.js.org/?q=verdaccio

**Comment by juanpicado** [comment]
another cool idea that might be explored
https://github.com/NodeSecure/cli
<img width="1498" alt="Screenshot 2022-06-28 at 13 36 08" src="https://user-images.githubusercontent.com/558752/176158995-925f04f1-dffd-4a86-b304-1104bd8ced56.png">
